python/GafferUI/PlugValueWidget.py

```diff
@@ -140,9 +140,9 @@ def getToolTip( self ) :
 
 		inputText = ""
 		if input is not None :
-			inputText = " &lt;- " + input.relativeName( input.commonAncestor( plug, Gaffer.GraphComponent ) )
+			inputText = " <- " + input.relativeName( input.commonAncestor( plug, Gaffer.GraphComponent ) )
 
-		result = "<h3>" + plug.relativeName( plug.node() ) + inputText + "</h3>"
+		result = "# " + plug.relativeName( plug.node() ) + inputText
```

Comment:
While we're at it, I think we should use inline code instead of headings for the plug names, since they're basically API references. To me, non-pretty plug names like `transform.pivot.z` just look wrong as headings. I'm aware the node Reference docs don't do this either, but it's something I should fix there too.

Reply:
They look wrong to me as headings too, but they look equally wrong to me as inline code, because when we put a bunch of text below them, they need to function as a heading in some way. The additional text we append to denote the input connection looks _really_ terrible as a heading though. I've experimented with a few alternatives but I'm not particularly happy with any of them. Can we deal with this as a followup including the reference docs? I haven't actually changed anything here...

Reply:
> I haven't actually changed anything here...

To be technical, it's gone from an H3 to an H1.

> Can we deal with this as a followup including the reference docs?

Absolutely.